specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,48 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the key, from your Azure KeyVault, you wish to use to protect your data. "
+        },
+        "keyVaultKeyVersion": {
+          "type": "string",
+          "description": "The specific version of the key, from your Azure KeyVault, you wish to use to protect your data."
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "description": "The URI of the Azure KeyVault where you created the key you want to use to protect your data. This value is also sometime referred to as the KeyVault DNS Name. An example value would be : https://your-keyvault-name.vault.azure.net."
```

Comment:
This is very helpful, and provides just the right amount of context.

A few minor wording, grammar and formatting nits:
- "the key you want to use to protect your data." -> "the key that you want to use to encrypt your data at rest."
- "sometime referred to" -> "sometimes referred to"
- "an example would be :" -> "an example would be:" (extra space before the `:`)